src/NHibernate.Test/BulkManipulation/HQLBulkOperations.cs

```diff
@@ -32,5 +32,23 @@ public void SimpleDelete()
 				tx.Commit();
 			}
 		}
+
+		protected override string MappingsAssembly => "NHibernate.Test";
+
+		protected override string[] Mappings => new[] { "BulkManipulation.Enrolment.hbm.xml" };
```

Comment:
The BaseFixture here will include all mappings set as embedded resources in its directory. There is no need for these overrides.

Reply:
Done, thanks for clarifying. I got an error for duplicate mappings after adding those (now reverted), and switching to TestCase (now reverted as well) addressed that.